app/build.gradle.kts

```
@@ -40,7 +44,9 @@ dependencies {
     implementation("androidx.appcompat:appcompat:1.6.0")
     implementation("com.google.android.material:material:1.7.0")
     implementation("androidx.constraintlayout:constraintlayout:2.1.4")
+    implementation(project(mapOf("path" to ":domain")))
```

```suggestion
    implementation(project(":domain"))
```
모듈 import 시 이렇게 작성하셔도 됩니다. 😄